src/main/scala/chisel3/util/util.scala

```diff
@@ -11,5 +11,6 @@ package object util {
   type ValidIO[+T <: Data] = chisel3.util.Valid[T]
   val ValidIO = chisel3.util.Valid
   val DecoupledIO = chisel3.util.Decoupled
-
+  val MixedVec = chisel3.util.HeterogeneousVec
+  val HetVec = chisel3.util.HeterogeneousVec
```

Comment:
I think we should settle on shorter name rather than just providing multiple

Reply:
Sure. I'm personally partial to `MixedVec`. I agree with you that `HVec` is overloaded and can be confusing, and `HetVec` still requires me as a user to stop and think about what is a "Het".

Reply:
`MixedVec` sounds good to me, should we discuss in the developers meeting tomorrow?

Reply:
Works for me, I think I would just replace all `HeterogeneousVec` with `MixedVec`